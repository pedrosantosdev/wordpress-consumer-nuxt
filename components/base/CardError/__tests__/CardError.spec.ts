import { describe, it, expect } from 'vitest'
import { render } from '@testing-library/vue'
import CardErrorVue from '@/components/base/CardError/CardError.vue'

describe('CardErrorVue', () => {
	it('should render properly', () => {
		const { html }= render(CardErrorVue)
		expect(html()).toMatchSnapshot()
	})
})
