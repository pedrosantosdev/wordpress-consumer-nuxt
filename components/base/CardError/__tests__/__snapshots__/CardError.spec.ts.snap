// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CardErrorVue > should render properly 1`] = `"<div class=\\"lg:w-96 w-full sm:h-64 md:h-auto flex flex-col sm:flex-row p-2 sm:p-4 rounded-2xl shadow-lg gap-5 select-none dark:bg-gray-800 bg-red-300 dark:bg-red-500 dark:text-white\\">Error</div>"`;
